In [1]:
import pandas as pd

In [18]:
cc_df = pd.read_csv("../data/raw/creditcard.csv")

# print(cc_df.info)# pretty useless
print(f"Shape - {cc_df.shape}, Size - {cc_df.size}")
print(f"{cc_df.columns}, Length - {len(cc_df)}")
print(f"Fradulent: {len(cc_df[(cc_df['Class'] == 1)])}")
print(f"Normal: {len(cc_df[(cc_df['Class'] == 0)])}")


Shape - (284807, 31), Size - 8829017
Index(['Time', 'V1', 'V2', 'V3', 'V4', 'V5', 'V6', 'V7', 'V8', 'V9', 'V10',
       'V11', 'V12', 'V13', 'V14', 'V15', 'V16', 'V17', 'V18', 'V19', 'V20',
       'V21', 'V22', 'V23', 'V24', 'V25', 'V26', 'V27', 'V28', 'Amount',
       'Class'],
      dtype='str'), Length - 284807
Fradulent: 492
Normal: 284315


There 492 fraudlent transactions present. The creator of the dataset indicates that it is 'highly unbalanced', so some class balancing techniques should be used (think imblearn).
Could be used in a classification model, predicting if transaction is fraudelent or not.
<br>
